semgrep/semgrep/config_resolver.py

```diff
@@ -124,7 +124,7 @@ def _nice_semgrep_url(self, url: str) -> str:
     def _download_config(self) -> Mapping[str, YamlTree]:
         config_url = self._config_path
         downloading_msg = (
-            f"Downloading config from {self._nice_semgrep_url(config_url)}..."
+            f"Downloading config from {self._nice_semgrep_url(config_url)} ..."
```

Comment:
@emjin might be good to have a comment so people don't think this is a typo

Reply:
what typo?